src/checkout/checkout-store-status-selector.ts

```diff
@@ -281,7 +282,16 @@ export default class CheckoutStoreStatusSelector {
     }
 
     /**
-     * Checks whether the current customer is updating their shipping address.
+     * Checks whether the shopper is continuing out as a guest.
+     *
+     * @returns True if continuing as guest, otherwise false.
+     */
+    isContinuingAsGuest(): boolean {
+        return this._billingAddress.isUpdating();
```

Comment:
So if the user is just updating their billing address, this method will return `true`? I think we might need to use a different key to track the status if we want to distinguish between the two.

Reply:
Yeah I was thinking about it. The way we queue up things mean we can't really have both things happening at the same time, but yeah I can see how in certain UIs it could be an issue.